server/paymentProviders/stripe/virtual-cards.js

```diff
@@ -216,6 +216,7 @@ export const processAuthorization = async (stripeAuthorization, stripeEvent) =>
         VirtualCardId: virtualCard.id,
         responsibleAdmin: responsibleAdmin.activity,
         collective: collective.activity,
+        expense,
```

Comment:
I'm surprised that we're using `expense`, not `expense.info here. Aren't we storing a huge payload in there? @SudharakaP could you post an example of `activiy.data.expense` as generated by this?

Reply:
Yes, this is my mistake. I should have used `expense.info` here. I've corrected that in the PR; 

The paypload for expense is,

```
Expense {
  dataValues: {
    createdAt: 2022-11-03T15:50:57.659Z,
    updatedAt: 2022-11-03T15:50:57.659Z,
    legacyPayoutMethod: 'manual',
    feesPayer: 'COLLECTIVE',
    id: 1,
    UserId: 5,
    CollectiveId: 7,
    FromCollectiveId: 5,
    currency: 'USD',
    amount: 100,
    description: 'Virtual Card charge: stripe',
    VirtualCardId: 'card_BM7mGwp1Ea8RtL',
    lastEditedById: 5,
    status: 'PROCESSING',
    type: 'CHARGE',
    incurredAt: 2022-11-03T15:50:57.650Z,
    data: { authorizationId: 'stripe_auth_id' },
    HostCollectiveId: null,
    payeeLocation: null,
    longDescription: null,
    PayoutMethodId: null,
    privateMessage: null,
    invoiceInfo: null,
    vat: null,
    deletedAt: null,
    tags: null,
    RecurringExpenseId: null
  },
  _previousDataValues: {
    UserId: 5,
    CollectiveId: 7,
    FromCollectiveId: 5,
    currency: 'USD',
    amount: 100,
    description: 'Virtual Card charge: stripe',
    VirtualCardId: 'card_BM7mGwp1Ea8RtL',
    lastEditedById: 5,
    status: 'PROCESSING',
    type: 'CHARGE',
    incurredAt: 2022-11-03T15:50:57.650Z,
    data: { authorizationId: 'stripe_auth_id' },
    id: 1,
    HostCollectiveId: null,
    payeeLocation: null,
    longDescription: null,
    legacyPayoutMethod: 'manual',
    PayoutMethodId: null,
    privateMessage: null,
    invoiceInfo: null,
    vat: null,
    feesPayer: 'COLLECTIVE',
    createdAt: 2022-11-03T15:50:57.659Z,
    updatedAt: 2022-11-03T15:50:57.659Z,
    deletedAt: null,
    tags: null,
    RecurringExpenseId: null
  },
  uniqno: 1,
  _changed: Set(0) {},
  _options: {
    isNewRecord: true,
    _schema: null,
    _schemaDelimiter: '',
    attributes: undefined,
    include: undefined,
    raw: undefined,
    silent: undefined
  },
  isNewRecord: false
}
```

whereas for `expense.info` it's,
```
{
  type: 'DEBIT',
  id: 1,
  UserId: 5,
  CollectiveId: 7,
  FromCollectiveId: 5,
  HostCollectiveId: null,
  currency: 'USD',
  amount: 100,
  description: 'Virtual Card charge: stripe',
  category: undefined,
  tags: null,
  legacyPayoutMethod: 'manual',
  vat: null,
  privateMessage: null,
  lastEditedById: 5,
  status: 'PROCESSING',
  incurredAt: 2022-11-03T15:52:16.080Z,
  createdAt: 2022-11-03T15:52:16.089Z,
  updatedAt: 2022-11-03T15:52:16.089Z
}
```